metasmoke.py

```diff
@@ -82,6 +82,7 @@ def init_websocket():
                 if not has_succeeded:
                     failed_connection_attempts += 1
                     if failed_connection_attempts > MAX_MS_WEBSOCKET_RETRIES:
+                        chatcommunicate.tell_rooms_with("debug", "Cannot initiate MS websocket. metasmoke_ws_t is now dead.")
```

Comment:
Most of the people in chat won't know or care what `metasmoke_ws_t` is, or what it means that it's dead. It would be be better to explain what this situation means to the people in chat. For example:

    "Cannot initiate MS WebSocket. A manual `!!/reboot` will be needed once MS is up."

Reply:
Would you think it is better to log metasmoke_ws_t while chat initiation-failed (or both log and chat initiation-failed)?

Reply:
In this instance, we should both `log` and `tell`. However, in this location, we shouldn't talk about `metasmoke_ws_t`, because the code here is completely independent from what the Thread happens to be called in ws.py. Also, we don't want to have to remember to change a string here if we happen to change a variable name that's used on two lines in another file.

In general, when looking through a log, it's going to be a person much more familiar with the code, and how programs work, than someone who's reading things in chat. I don't know about other people, but in the log, I'm looking at cues as to where the code is currently operating and its state; mentally going through the process of running the code as I scan through what's there.

In chat, people are not expected to know the details of the code. What helps them is to know the status of SD and how a change of status affects them (i.e. what they might need to do).